### PR TITLE
Add file command for rsession-complete images

### PR DESCRIPTION
**Pull request by jeffvroom**
The rsession will try to run `file ...` when trying to open a file without a known suffix.  If that command gets an error, it reports the file as binary so we need this extra command.

### CODE REVIEW

**Review by melissa-barca — APPROVED** [review]
LGTM

**Review by ianpittwood — APPROVED** [review]

**Review by cm421 — APPROVED** [review]
LGTM!

File: r-session-complete/Dockerfile.ubuntu2204 [hunk 1]
```diff
@@ -27,6 +27,7 @@ RUN apt-get update \
       libpq-dev \
       rrdtool \
       subversion \
+      file
```
Comment (ianpittwood) [inline r-session-complete/Dockerfile.ubuntu2204:30]:
```suggestion
      file \
```

**Review by ianpittwood — CHANGES_REQUESTED** [review]
My bad, we missed one `\` here. Other than that I think it should be good pending the builds.

**Review by colearendt — APPROVED** [review]
LGTM! Thanks @jeffvroom ! This is an interesting problem - have y'all thought about whether these types of dependencies should be declared explicitly (i.e. as a build / install dependency?). It's weird because it isn't a proper system dependency, but we do shell out to something that we depend on on the system 🤷‍♂️

**Comment by jeffvroom** [comment]
@colearendt Thanks, that's a great suggestion. How about if I cancel this PR and fix this problem in:  https://github.com/rstudio/rstudio-pro/issues/4455?

**Pull request closed without merging**